julia-mode/while

```diff
@@ -0,0 +1,9 @@
+# -*- mode: snippet -*-
+# name: while ... ... end
+# key: while
+# contributor: hankail05
+# --
+while ${1:cond}
+    ${2:body}
```

Comment:
Please add \`%\` before `${2:body}` to give this snippet implicit selection support.

```diff
-  ${2:body}
+  `%`${2:body}
```